cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -136,7 +136,12 @@ public KafkaBrokerConfigurationBuilder withListeners(String clusterName, String
 
         // Replication listener
         listeners.add("REPLICATION-9091://0.0.0.0:9091");
-        advertisedListeners.add(String.format("REPLICATION-9091://%s-${STRIMZI_BROKER_ID}.%s-brokers.%s.svc:9091", KafkaResources.kafkaStatefulSetName(clusterName), KafkaResources.kafkaStatefulSetName(clusterName), namespace));
+        advertisedListeners.add(String.format("REPLICATION-9091://%s:9091",
+                ModelUtils.podDnsName(namespace,
```

Comment:
Should this be `podDnsNameWithoutClusterDomain`?

Reply:
You're absolutely right, I made these changes prior to our conversation and misunderstood. whether they should be long or short.